ext/ddtrace.h

```diff
@@ -64,6 +64,10 @@ void dd_internal_handle_fork(void);
 void dd_run_rust_thread_destructors(void *unused);
 #endif
 
+#ifndef _WIN32
+void disable_sidecar_sending(void);
```

Comment:
nitpick: this function does not seem to be used elsewhere but in `ddtrace.c`. So you could declare it before its usage (+ make it `static`) and remove the declaration from here